lib/use_case/count_views.rb

```diff
@@ -0,0 +1,40 @@
+module UseCase
+  class CountViews
+    def initialize(file_gateway:, count_strategy:, order_strategy:)
+      @file_gateway = file_gateway
+      @count_strategy = count_strategy
+      @order_strategy = order_strategy
+    end
+
+    def execute(presenter:)
+      visits = sort(count(hash_of_visits))
```

Comment:
Using dependency injection (DI) allows us to construct use cases specific to our needs:
* from where are we getting file? `file_gateway`
* do we want unique count or all is fine? `count_strategy`
* what should be the sorting order? `order_strategy`
* do we want a response for an API or to print in the console? `presenter`